pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, fetchzip, SDL, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  pname = "hex-a-hop";
+  version = "1.1.0";
+
+  src = fetchzip {
+    url = "https://downloads.sourceforge.net/project/hexahop/${version}/hex-a-hop-${version}.tar.gz";
+    sha256 = "sha256-fBSvNtgNR0aNofbvoYpM1e8ww4ARlXIvrQUvJqVGLlY=";
+  };
+
+  icon = fetchurl {
+    url = "https://sources.debian.org/data/main/h/hex-a-hop/${version}%2Bgit20140926-1.1/data/hex-a-hop.png";
+    sha256 = "sha256-Vh/1wwRmC2eSD/+mk1Oqt7EX4a4k++nUAbWQD2P2hNA=";
+  };
+
+  buildInputs = [ SDL SDL_mixer SDL_ttf ];
+
+  nativeBuildInputs = [ copyDesktopItems ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  desktopItems = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    desktopName = pname;
+    comment = meta.description;
+    type = "Application";
+    categories = [ "Game" "LogicGame" ];
+    startupNotify = false;
+  };
+
+  postFixup = ''
+    install -Dm644 ${icon} $out/share/icons/${pname}.png
+  '';
+
+  meta = {
+    description = "A puzzle game based on hexagonal tiles";
+    homepage = "http://hexahop.sourceforge.net";
+    license = lib.licenses.gpl2Plus;
```

Comment:
Where does it say that?

Also, some of the sound files have `nonfree` in their name. What's up with that? Are the assets truly under GPLv2?

Reply:
Yes, they are. The licenses are on the xml files: https://salsa.debian.org/games-team/hex-a-hop/-/blob/master/data/music-ending.ogg.xml

Debian gives detailed information for the Copyrights https://metadata.ftp-master.debian.org/changelogs//main/h/hex-a-hop/hex-a-hop_1.1.0+git20140926-1.1_copyright

The COPYING file contains the license.

Reply:
About the non-free ones https://sourceforge.net/p/hexahop/code/ci/3f654d9d253f68aa1b312125aad9ad9daff14e0d/

Reply:
Please add CC-BY-SA 3.0 to the licenses then.

Reply:
Sure, I only added the main one because that's what most packages on nixpkgs seem to be doing.
Should I add the other ones as listed by Debian? there's also some LGPL-2+ and LGPL-3+ files.

Reply:
Add all licenses that apply to the packaged end result. Whether it's code or assets doesn't matter.

Reply:
Added.